test/TriGParserTest.php

```diff
@@ -69,6 +69,9 @@ public function shouldNotParse($createParser, $input, $expectedError = null): vo
                 $this->fail("Expected this error to be thrown (but it wasn't): ".$expectedError);
             }
         });
+        if (!$errorReceived) {
```

Comment:
```suggestion
        if (false === $errorReceived) {
```
Just a small thing, but its more obvious what we check.

Reply:
Ufff, I disagree with testing whether a boolean is a certain boolean